### PR TITLE
deepsource fixes

### PR DESCRIPTION
**Pull request by michal367**
fixes:
- Consider using in - PYL-R1714
- Unnecessary parentheses after keyword - PYL-C0325
- Inconsistent return statements - PYL-R1710
- Audit: Starting a process with a partial executable path - BAN-B607
- Possible shell injection via Paramiko call - BAN-B601
- Audit required: Use of eval - PYL-W0123
- Too many positional arguments in function call - PYL-E1121
- Missing argument in function call - PYL-E1120
- Unnecessary else / elif used after break - PYL-R1723
- Unnecessary elif / else block after continue - PYL-R1724
- Unnecessary else / elif used after return - PYL-R1705
- Unnecessary else/elif used after raise - PYL-R1720
- Loop variable used outside the loop - PYL-W0631
- Exception caught is very general - PYL-W0703
- Comparison with itself - PYL-R0124
- Argument redefined from local - PYL-R1704
- Re-definition found for builtin function - PYL-W0622
- Implicit enumerate calls found - PTC-W0060
- if statements can be merged - PTC-W0048
- Function/method with an empty body - PTC-W0049
- File opened without the with statement - PTC-W0010
- Unused variable found - PYL-W0612
- Dangerous default argument - PYL-W0102

### CODE REVIEW

**Review by grzanka — APPROVED** [review]
Looks OK

**Comment by grzanka** [comment]
Fix as well this one:
image

**Review by Yoanka — CHANGES_REQUESTED** [review]

File: pytrip/res/concave_tool.py [hunk 1]
```diff
@@ -1,307 +1,326 @@
...
+        2.2. Connect parts of contour (taking into account direction of connection)
+    3. Return created contours.
+Basic heuristics are used:
+    1. searching for minimal distance between a part of contour and a point and ensuring that relation is symmetrical
+        (if point A is the closest one to part B, check if part B is the closest one to point A)
+    2. limiting distance between parts and points based on previous distances
+        (next point cannot be too far away from parts)
+"""
+
+import math
```
[the remark this answers is not in the document]
Comment (michal367) [reply pytrip/res/concave_tool.py:34]: I've moved import, added license comment at the beginning of the file and converted CRLF to LF.

**Review by Yoanka — APPROVED** [review]
Looks good to me.

**Comment by grzanka** [comment]
@michal367 could you please squash-merge ?

**Pull request merged**